google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1alpha2/ProtoSchemaConverterTest.java

```diff
@@ -159,4 +152,27 @@ public void convertRecursive() {
       // Expected exception
```

Comment:
Could assertEquals(e.getMessage(), "Recursive type is not supported: <fullName>" )